packages/big-design/src/components/List/List.tsx

```diff
@@ -83,6 +90,63 @@ const StyleableList = typedMemo(
       scheduleUpdate();
     }, [isOpen, height, width, selectedItems?.length]);
 
+    const handleSelectAll = useCallback(
+      (listItems: Array<SelectOption<T>>) => {
+        if (updateItems) {
+          const enabledItems = listItems.filter((item) => !item.disabled);
+
+          updateItems(enabledItems);
+        }
+      },
+      [updateItems],
+    );
+
+    const handleUnselectAll = useCallback(() => {
+      updateItems && updateItems([]);
```

Comment:
the comment applied to all cases where we had an expression that wasn't really used. eg `a && a()` not just the one I pointed out

Reply:
My bad, missed that

Already fixed. Thanks.